src/Util/util.php

```diff
@@ -22,18 +20,19 @@
  * @param  string $ps   The string to use as the path separator.
  * @return string       The relative path between $from and $to.
  */
-function rel_path( $from, $to, $ps = PS ): string {
-	// Turn paths into array.
-	$arFrom = explode($ps, rtrim( normalize( $from, $ps ), $ps ) );
-	$arTo = explode( $ps, rtrim( normalize( $to, $ps ), $ps ) );
-	// Strip the common roots from both arrays.
-	while( count( $arFrom ) && count( $arTo ) && ( $arFrom[0] == $arTo[0] ) ) {
-		array_shift( $arFrom );
-		array_shift( $arTo );
-	}
-	// for any items left in from, add '../' and then append the remaining
-	// to items.
-	return str_pad( '', count( $arFrom ) * 3, '..' . $ps ) . implode( $ps, $arTo );
+function rel_path($from, $to, $ps = DIRECTORY_SEPARATOR): string
+{
```

Comment:
Did you put this `{` on a newline accidentally?

Reply:
Oh, they all are ... looks like this is on purpose then :)

Reply:
Yeah added php-cs-fixer and set it to PSR-12 (https://github.com/boxuk/wp-muplugin-loader/pull/21/files#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0R12)